cmd/oras/attach.go

```diff
@@ -44,7 +44,7 @@ type attachOptions struct {
 func attachCmd() *cobra.Command {
 	var opts attachOptions
 	cmd := &cobra.Command{
-		Use:   "attach name<:tag|@digest> [file[:type]...]",
+		Use:   "attach [flags] <name>{:<tag>|@<digest>} {<file>[:<type>]}[...]",
```

Comment:
Can we close #603 and merge the change here? Also, update the description to resolve #602?
```suggestion
		Use:   "attach [flags] --artifact-type=<type> <name>{:<tag>|@<digest>} [<file>[:<type>]] [...]",
```